decidim-core/db/migrate/20180209122819_create_decidim_searchable_rsrcs.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class CreateDecidimSearchableRsrcs < ActiveRecord::Migration[5.1]
+  def change
+    create_table :decidim_searchable_rsrcs do |t|
+      t.text :content_a
+      t.text :content_b
+      t.text :content_c
+      t.text :content_d
+      t.string :locale, null: false
+      t.datetime :datetime
```

Comment:
What does this attribute mean? `:datetime` as a name of the attribute seems very confusing to me

Reply:
datetime:  The timestamp that places this resource in the line of time. Used as second criteria (first is text relevance) for sorting.

Reply:
Yeah, but then again we could have fields called `:integer`, `:string`, or `:text`...

Reply:
Sorry I didn't saw this comment. What name do you like best? `timestamp`? `at`? other proposal? It is a difficult choice because depending on the indexed model it will have different meaning.